modules/currency-l0/src/main/scala/org/tessellation/currency/l0/CurrencyL0App.scala

```diff
@@ -115,7 +115,7 @@ abstract class CurrencyL0App(
         .handlers <+>
         services.consensus.handler
       _ <- Daemons
-        .start(storages, services, programs, queues, healthChecks, services.dataApplication, cfg)
+        .start(storages, services, programs, queues, healthChecks, services.dataApplication, cfg, method.nodeSharedConfig.doubleSignDetect)
```

Comment:
`isn't the config already passed through the `cfg`?

Reply:
Moved all configuration for the daemon to `DoubleSignDetectDaemonConfig`, which are included in the various `AppConfig`s.